src/DryIoc/Container.cs

```diff
@@ -11353,7 +11353,10 @@ public static ParameterSelector Details(this ParameterSelector source, Func<Requ
     public static partial class PropertiesAndFields
     {
         /// <summary>Say to not resolve any properties or fields.</summary>
-        public static PropertiesAndFieldsSelector Of = request => null;
+        public static PropertiesAndFieldsSelector Off = request => null;
```

Comment:
@michkot Hi, and first thank you for the PR. 

Second here, I don't like the `Of` and `Off` change here. Yes, I do think that it may be confusing, but introducing `Off` which is the same as `Of` does not resolve the problem, and moreover makes it a bit more confusing.

In regard to `typo in name.` comment - it was not a typo. The `Of` name is intentional and consistently used for other things, e.g. `Made.Of` etc.

I suggest reverting those changes and I will merge the fix to `All` comments.